plugin/al/usdmayautils/AL/usdmaya/utils/MeshUtils.cpp

```diff
@@ -2072,7 +2174,7 @@ void MeshExportContext::copyNormalData(UsdTimeCode time, bool copyAsPrimvar)
           bool isOrdered = true;
           for(uint32_t i = 0, n = normalIndices.length(); i < n; ++i)
           {
-            if(uint32_t(normalIndices[i]) != i)
+            if(normalIndices[i] != i)
```

Comment:
error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
`if(normalIndices[i] != i)`